src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/scheduler/BaseTask.java

```diff
@@ -65,41 +63,52 @@ public void run() {
                         ccdService.getCaseByCaseId(caseReference.getCaseReference(), getCaseType(), systemUserToken);
                     log.info("SearchResult count {}", searchResult.getTotal());
                     if (CollectionUtils.isNotEmpty(searchResult.getCases())) {
+                        if (!isUpdatedRequired(searchResult.getCases().getFirst())) {
+                            log.info("No update required for case reference {}", caseReference.getCaseReference());
+                            continue;
+                        }
+
                         StartEventResponse startEventResponse = ccdService.startEventForCaseWorker(systemUserToken,
                             caseReference.getCaseReference(), getCaseType().getCcdType(), EventType.AMEND_CASE_CRON.getCcdType());
 
                         CaseDetails caseDetails = startEventResponse.getCaseDetails();
                         FinremCaseDetails finremCaseDetails = finremCaseDetailsMapper.mapToFinremCaseDetails(caseDetails);
                         log.info("Updating {} for Case ID: {}", getTaskName(), caseDetails.getId());
                         executeTask(finremCaseDetails);
+                        String description = getDescription(finremCaseDetails);
                         CaseDetails updatedCaseDetails = finremCaseDetailsMapper.mapToCaseDetails(finremCaseDetails);
                         startEventResponse.getCaseDetails().setData(updatedCaseDetails.getData());
                         ccdService.submitEventForCaseWorker(startEventResponse, systemUserToken,
                             caseDetails.getId().toString(),
                             getCaseType().getCcdType(),
                             EventType.AMEND_CASE_CRON.getCcdType(),
                             getSummary(),
-                            getSummary());
+                            description);
                         log.info("Updated {} for Case ID: {}", getTaskName(), caseDetails.getId());
                     }
-
                 } catch (InterruptedException | RuntimeException e) {
-                    log.error("Error processing caseRef {} and error is ", caseReference.getCaseReference(), e.getMessage());
-                    e.printStackTrace();
+                    log.error("Cron task {}: Error processing case {}", getTaskName(), caseReference.getCaseReference(), e);
                 } finally {
                     RequestContextHolder.resetRequestAttributes();
                 }
-
             }
         }
     }
 
-
     protected String getSystemUserToken() {
         log.info("Getting system user token");
         return systemUserService.getSysUserToken();
     }
 
+    /**
+     * Check to determine if the case needs to be updated by the task.
+     * @param caseDetails the case to check.
+     * @return true if the case needs to be updated, false otherwise.
+     */
+    protected boolean isUpdatedRequired(CaseDetails caseDetails) {
+        return true;
+    }
```

Comment:
Looks like you meant to actually return the result based on the passed case. Just checking.